dandiapi/api/tasks/__init__.py

```diff
@@ -139,7 +139,7 @@ def validate_version_metadata(version_id: int) -> None:
 
         validate(metadata, schema_key='PublishedDandiset', json_validation=True)
     except dandischema.exceptions.ValidationError as e:
-        logger.info('Error while validating version %s', version_id)
+        _log_version_action('Error while validating dandiset metadata', version_id, version)
```

Comment:
@waxlamp I have rebased and added this extra log line. But (attn @AlmightyYakob ) shouldn't it have been `logger.error` for a better use of logging facility? (then I would adjust the helper to accept optional level option and use logging.ERROR level)

Reply:
Version metadata failing validation is an expected code path, so imo using `logger.error` here wouldn't be appropriate.

Reply:
makes thanks, thanks!